dashboard/src/server-config/ci.ts

```diff
@@ -10,4 +10,10 @@ const ci: ConfigInterface = {
   },
 };
 
+if (process.env.STRIPE_SECRET_KEY)
+  ci.stripe = {
+    secretKey: process.env.STRIPE_SECRET_KEY,
+    paymentLink: 'https://buy.stripe.com/test_bIY3fy7X4bzz4Io000',
```

Comment:
are we ok with exposing the link in CI, or we should go with a secret?

Reply:
yes, this is a totally safe/public link